server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -558,7 +560,7 @@ private void setupDefaultSecurityLabels(KeyExtent extent) {
             tableConfiguration.get(Property.TABLE_DEFAULT_SCANTIME_VISIBILITY));
         this.defaultSecurityLabel = cv.getExpression();
       } catch (Exception e) {
-        log.error("{}", e.getMessage(), e);
+        log.error("Error setting up default security label " + e.getMessage(), e);
```

Comment:
```suggestion
        log.error("Error setting up default security label {}", e.getMessage(), e);
```

Reply:
This will drop the stack trace.  Sl4j doesn't have a method that allows wrapping the exception and using the format string.

Reply:
If the last element of the array is a Throwable, it will work.

Reply:
Yeah you are right.  Looking at the impl of Log4j at least, it does that.  Cool, thanks.